File: cash.c
#include<stdio.h>
#include<cs50.h>
#include<math.h>
int coin(int n);
int main(void)
{
    float d;
    int c;
do
  {
     d=get_float("change owed:");
     c=round(d*100);
          }
     while (c<1);
     printf("%i\n",coin(c));
}
int coin(int n)
{     int co=0;
    while(n>0)
    {
        if(n>=25)
        {
            n=n-25;
            co=co+1;
        }
        else if(n<25 && n>=10)
        {
            n=n-10;
            co=co+1;
        }
        else if(n<10 && n>=5)
        {
            n=n-5;
            co=co+1;
        }
        else
        {
            n=n-1;
            co=co+1;
        }
    }
    return co;
} 

File: cash/cash.py
from cs50 import get_float

def main():
    n = get_positive_int()
    noofcoin(n)

def get_positive_int():
    while(True):
        m = get_float("change owed: ")
        if m > 0:
            break
    return m

def noofcoin(n):
    x = round(n * 100)
    count = 0
    while(x > 0):
        if x >= 25:
            x -= 25
            count += 1
        elif x >= 10:
            x -= 10
            count += 1
        elif x >= 5:
            x -= 5
            count += 1
        else:
            x -= 1
            count += 1

    print(count)

main()
